nss-day-cohort-60/django-taters-api
# Nv list retrieve posts

Closes #19 

# Description of Proposed Changes
User can get all posts and single posts with serialized manytomany relationships; Reactions and Tags 
The server should sent appropriate status codes in each

---


# Steps to Test

1. Fetch the `nv-list-retrieve-posts` branch and switch to it
2. make a GET request in postman to `http://localhost:8000/posts` 
3. verify that response body contains all posts data including Reactions and Tags
4. verify response code is 201
5. make a GET request in postman to `http://localhost:8000/posts/1` 
6. steps 3-4
